finger/models.py

```diff
@@ -1,5 +1,29 @@
 from django.contrib.auth.models import AbstractUser
 from django.db import models
+import re
+
+class UserManager(models.Manager):
+    def update_data(self, data):
+        """
+        Take an iterable of user data (the result of parsing finger.json),
+        and create and/or update users accordingly.
+        """
+        for entry in data:
+            username = entry.get('användarnamn')
+            if username and re.match('^[a-z]+$', username) and username != 'stacken':
+                fields = {
+                    'first_name': entry.get('förnamn'),
+                    'last_name': entry.get('efternamn'),
+                    'email': entry.get('mailadress'),
+                }
+                if not fields.get('email'):
+                    t = entry.get('KTH-konto')
```

Comment:
I'm thinking this variable could probably have a slightly more descriptive name than `t`, because I don't like single-character variable names.

Reply:
True.  In other languages, I tend to be ok with single-chars variable names if the scope is small enough, but in python the scope is allways the method anyway, and that's not really short in this case.  :-)